src/contribute/contribution-guide.md

```diff
@@ -231,10 +235,54 @@ Then you can push your local, committed changes to your (forked) repository on G
 ### Testing
 All code should have appropriate unit testing coverage. New code should have new tests in the same contribution. Bug fixes should include a regression test to prevent the issue from reoccurring.
 
+#### Java SDK
+
 For contributions to the Java code, run unit tests locally via Maven. Alternatively, you can use Travis-CI.
 
     $ mvn clean verify
 
+#### Python SDK
+
+For contributions to the Python code, you can use following commands to run unit tests locally and to check for lint errors. Once you send a pull request for review, these tests will be rerun using Jenkins. We recommend setting up a virtual environment before testing your code.
```

Comment:
Don't follow -- which are "following commands"?

Fix above: Travis CI is disabled.
Overall, I'd probably generalize and say that mvn clean verify does it all, i.e., no Java SDK/Python SDK sections at all.